juniper_official/routing/collect-rsvp-stats.rule

```diff
@@ -0,0 +1,200 @@
+/*
+ * Collect RSVP LSP traffic statistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * One input control detection
+ *
+ *   1) input-lsp-name, is a regular expression that matches the lsp name
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all lsps. Use something like 'Silver*' to match only lsp names starting
+ *      with 'Silver'. 
+ *
+ */
+ healthbot {
+    topic routing.mpls {
+        rule collect-rsvp-stats {
+            keys lsp-name;
+            description "Collects LSP stats periodically";
+            sensor lsp {
+                open-config {
+                    sensor-name /junos/services/label-switched-path/usage/;
```

Comment:
In already existing rsvp rule "
https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/routing/check-rsvp-neighbor-state.rule
" a different sensor path is used. Is there a way that we can use the sensor path for both the rules? [6:40 pm] Vijay Gadde
Or in these cases the these sensors send completely different data?